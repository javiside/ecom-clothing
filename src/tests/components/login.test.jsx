import React from "react";
import { shallow } from "enzyme";

import Login from "../../components/login";

describe("Login", () => {
  it("should mount the component correctly", () => {
    expect(shallow(<Login />)).toMatchSnapshot();
  });
});
