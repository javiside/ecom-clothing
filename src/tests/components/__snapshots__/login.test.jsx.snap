// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should mount the component correctly 1`] = `
<div
  className="login"
>
  <h2
    className="heading"
  >
    I already have an account
  </h2>
  <span
    className="title"
  >
    Login with your email and password
  </span>
  <span
    className="login-error"
  />
  <form
    onSubmit={[Function]}
  >
    <FormInput
      handleChange={[Function]}
      label="email"
      name="email"
      required={true}
      type="email"
      value=""
    />
    <FormInput
      autoComplete="on"
      handleChange={[Function]}
      label="password"
      name="password"
      required={true}
      type="password"
      value=""
    />
    <div
      className="buttons"
    >
      <CustomButton
        type="submit"
      >
        LOGIN
      </CustomButton>
      <CustomButton
        isGoogleSignIn={true}
        onClick={[Function]}
        type="button"
      >
        Sign in with Google
      </CustomButton>
    </div>
  </form>
</div>
`;
